ui/desktop/electron-app/src/index.js

```diff
@@ -113,11 +113,18 @@ app.on('ready', async () => {
   }
   Menu.setApplicationMenu(menuTemplate);
 
+  /**
+   * Enable electron OS window frame/chrome for MacOS only.
+   * Disable frame/chrome regardless of OS when `DISABLE_WINDOW_CHROME` is set.
+   */
+  let showWindowChrome = isMac();
```

Comment:
We should plan a way to move platform-dependent configuration into a separate file and set relevant values there, rather than inline with our normal code.  But that can happen in a separate PR.